common/src/main/java/eu/midnightdust/lib/config/MidnightConfig.java

```diff
@@ -85,6 +85,25 @@ public <T> void writeList(int index, T value) {
         } return null;
     }
 
+    public static boolean checkRequirements(Requires req) {
+        String reqString = req.requirement();
+        Field reqField;
+        try {
+             reqField = req.requirementSource().getDeclaredField(reqString);
+        } catch (NoSuchFieldException e) {
+            throw new RuntimeException("Field does not exist!", e);
```

Comment:
I'm not a fan of hard exceptions. 
We can just return false here and leave an error message in the log.